benchmarks/squad/README.md

```diff
@@ -0,0 +1,34 @@
+
+The goal is to provide an answer which matches between 1 to 3 of the target answers, in response to a question based on a paragraph from a provided Wikipedia article. 
+
+The questions are designed to vary lexically and syntactically from the provided paragraphs, and thus, answering them requires advanced reasoning capabilities.
+
+## Execution
+Here is an example from the dataset:
+```
+Context: Super Bowl 50 was an American football game to determine the champion of the National Football League (NFL) for the 2015 season. The American Football Conference (AFC) champion Denver Broncos defeated the National Football Conference (NFC) champion Carolina Panthers 24–10 to earn their third Super Bowl title. The game was played on February 7, 2016, at Levi's Stadium in the San Francisco Bay Area at Santa Clara, California. As this was the 50th Super Bowl, the league emphasized the "golden anniversary" with various gold-themed initiatives, as well as temporarily suspending the tradition of naming each Super Bowl game with Roman numerals (under which the game would have been known as "Super Bowl L"), so that the logo could prominently feature the Arabic numerals 50.
+
+Question: If Roman numerals were used, what would Super Bowl 50 have been called?
+
+Answer(s): Super Bowl L, L, Super Bowl L
+```
+The model is tasked to answer the question by referring to the context, potentially requiring mutliple-sentence reasoning.
+
+## Evaluation
+The model is prompted with the Wikipedia paragraph acting as the context, and the corresponding question. It is required to answer the question, only using the context provided and no outside information. The prompt template takes inspiration from the following OpenAI examples:
+- [fine_tuned_qa example](https://github.com/openai/openai-cookbook/blob/627a11cb2f2c7a174c42c724c2e8a9737f79e6e1/examples/fine-tuned_qa/ft_retrieval_augmented_generation_qdrant.ipynb)
+- [unsolvable_questions example](https://github.com/openai/evals/blob/234bcde34b5951233681455faeb92baaaef97573/evals/registry/data/unsolvable_questions/convert.js)
+
+The evaluation performed, in particular the F1-score calculation logic, draws from the official SQuAD v2 [evaluation script](https://storageclwsprod1.blob.core.windows.net/bundles/0x6b567e1cf2e041ec80d7098f031c5c9e/contents.gz?se=2024-08-27T20%3A03%3A20Z&sp=rt&sv=2019-12-12&sr=b&rscd=inline%3B%20filename%3D%22evaluate-v2.0.py%22&rsce=gzip&rsct=text/x-python&sig=IJaOf0A7VbZgIaYMe6YdLYu8aN6LPHHFkUvLAOSt%2BQ0%3D), as well as EleutherAI's lm-evaluation-harness for the [DROP benchmark](https://github.com/EleutherAI/lm-evaluation-harness/blob/main/lm_eval/tasks/drop/utils.py#L64C1-L73C40) and the [SuperGLUE benchmark](https://github.com/EleutherAI/lm-evaluation-harness/blob/ebe7226ebfb8d11a9fb8d6b53eb65891f895c633/lm_eval/tasks/super_glue/record/t5_utils.py).
```

Comment:
I get AuthenticationFailed error when I click on the first link.

Reply:
Could you check please if the new link works for you?

Reply:
Same, it might be restricted in my location. Could you ask your peer to check as well?

Reply:
I think this is out-of-date, I've updated it to this: https://worksheets.codalab.org/rest/bundles/0x6b567e1cf2e041ec80d7098f031c5c9e/contents/blob/